.circleci/config.yml

```diff
@@ -828,6 +828,22 @@ jobs:
           path: tmp/capybara
           destination: capybara
 
+  portafly:
+    resource_class: medium+
+    docker:
+      - image: circleci/node:10
```

Comment:
Why not using a more recent version of node? I don't think we need to productize it.

Reply:
We could, yeah. Just was matching the Node version in Porta. What do you reckon @hallelujah ?

Reply:
I am OK on upgrading everything (at the same time) but you know the boilerplate we have.